about/about-kubernetes-services.md

```diff
@@ -0,0 +1,147 @@
+the backing pods themselves do not see the client IP, with the same implications for network policy.  As with node
+ports, this limitation can be circumvented in some scenarios by using [externalTrafficPolicy](#externaltrafficpolicylocal)
+or by using {{site.prodname}}'s eBPF dataplane [native service handling](#calico-ebpf-native-service-handling) (rather
+than kube-proxy) which preserves source IP address.
+
+### Advertising service IPs
+
+One alternative to using node ports or network load balancers is to advertise service IP addresses over BGP. This
+requires the cluster to be running on an underlying network that supports BGP, which typically means an on-prem
+deployment with standard Top of Rack routers.
+
+{{site.prodname}} supports advertising service Cluster IPs, or External IPs for services configured with one. If you are
+not using Calico as your network plugin then {% include open-new-window.html text='MetalLB'
+url='https://github.com/metallb/metallb' %} provides similar capabilities that work with a variety of different network
+plugins.
+
+![kube-proxy service advertisement]({{site.baseurl}}/images/kube-proxy-service-advertisement.svg)
+
+### externalTrafficPolicy:local
+
+By default, whether using service type `NodePort` or `LoadBalancer` or advertising service IP addresses over BGP,
+accessing a service from outside the cluster load balances evenly across all the pods backing the service, independent
+of which node the pods are on. This behavior can be changed by configuring the service with
+`externalTrafficPolicy:local` which specifies that connections should only be load balanced to pods backing the service
+on the local node.
+
+When combined with services of type `LoadBalancer` or with {{site.prodname}} service IP address advertising, traffic is
+only directed to nodes that host at least one pod backing the service. This reduces the potential extra network hop
+between nodes, and perhaps more importantly, to maintain the source IP address all the way to the pod, so network policy
+can restrict specific external clients if desired.
+
+![kube-proxy service advertisement]({{site.baseurl}}/images/kube-proxy-service-local.svg)
+
+Note that in the case of services of type `LoadBalancer`, not all Load Balancers support this mode. And in the case of
+service IP advertisement, the evenness of the load balancing becomes topology dependent. In this case, pod anti-affinity
+rules can be used to ensure even distribution of backing pods across your topology, but this does add some complexity to
+deploying the service.
+
+### Calico eBPF native service handling
+
+As an alternative to using Kubernetes standard kube-proxy, {{site.prodname}}'s eBPF dataplane supports native service
+handling. This preserves source IP to simplify network policy, uses DSR (Direct Server Return) to reduce the number of network hops for return traffic, and provides even load balancing independent of topology, with reduced CPU and latency compared to kube-proxy.
```

Comment:
Could say `offers DSR` instead of `uses DSR` since it's not available everywhere.